tensorflow/python/training/monitored_session.py

```diff
@@ -267,7 +267,7 @@ def get_or_default(arg_name, collection_key, default_constructor):
     return op
 
   @staticmethod
-  def _default_local_init_op():
+  def default_local_init_op():
     return control_flow_ops.group(
```

Comment:
Please add a docstring.

Reply:
Ideally, say which initialization steps this method includes, in which order, so it's possible to use it without looking at the code.